app/javascript/chat/components/ChannelFilterButton.jsx

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { Button } from '@crayons';
+
+export const ChannelFilterButton = ({ type, name, active, onClick }) => {
+  return (
+    <Button
+      data-channel-type={type}
+      data-testid={name}
+      onClick={onClick}
+      className={`chat__channeltypefilterbutton crayons-indicator crayons-indicator--${
+        type === active ? 'accent' : ''
```

Comment:
as per my other comment - if we change the `active` prop to a boolean, we could replace this with 
```suggestion
        active ? 'accent' : ''
```